### PR TITLE
Hp training faker

### PR DESCRIPTION
**Pull request by hunterphillips**
Adds training programs to database and to build-db function 

Running npm run build-test should successfully build an sqlite db with: customers, computers, programs, and departments

Relates to #9 and #23

### CODE REVIEW

**Review by melissabell456 — CHANGES_REQUESTED** [review]
attendees is incorrectly spelled

**Review by dhazlettjr — APPROVED** [review]
All Clear

**Review by electriceeli — APPROVED** [review]
I will allow it

File: package.json [hunk 1]
```diff
@@ -22,6 +22,6 @@
     "faker": "^4.1.0"
   },
   "dependencies": {
-    "sqlite3": "^4.0.0"
+    "sqlite3": "^3.1.9"
```
Comment (Valerie-Freeman) [inline package.json]: Please change sqlite3 version to 4.0.0

**Comment by hunterphillips** [comment]
Fixed attendees typo and sqlite3 version

**Comment by melissabell456** [comment]
thanks! good to go.

**Pull request merged**